src/samples/public-client-app-sample/pom.xml

```diff
@@ -20,7 +20,7 @@
 		<dependency>
 			<groupId>com.nimbusds</groupId>
 			<artifactId>oauth2-oidc-sdk</artifactId>
-			<version>4.5</version>
+			<version>5.24.1</version>

```

Comment:
>version>5.24.1</version> [](start = 4, length = 24)

is this the approved version?

Reply:
switch to 5.24.1 happened with 1.3.0 release

Reply:
ack. but is this the version which you referenced in the OSS tool?

---
In reply to: [179283962](https://github.com/AzureAD/azure-activedirectory-library-for-java/pull/205#discussion_r179283962) [](ancestors = 179283962)

Reply:
Yes, 5.24.1 is referenced